jsonrpclib/SimpleJSONRPCServer.py

```diff
@@ -1,19 +1,26 @@
 import jsonrpclib
 from jsonrpclib import Fault
 from jsonrpclib.jsonrpc import USE_UNIX_SOCKETS
-import SimpleXMLRPCServer
-import SocketServer
+try:
```

Comment:
Sounds good. Do you think the names should be "modernized" throughout the code too? ie. SimpleXMLRPCServer becomes xmlrpc.server, SocketServer becomes socketserver? I originally tried to keep changes as minimal as possible
Btw, I'm quite new to github so apologies in advance if I make any process sorts of mistakes.